integration/solana/setup.ts

```diff
@@ -117,12 +117,29 @@ async function setup() {
 
             console.log(`Loading ${name} at ${program.publicKey}...`);
             const program_so = fs.readFileSync(file);
-            await BpfLoader.load(connection, payer, program, program_so, BPF_LOADER_PROGRAM_ID);
+            for (; ;) {
```

Comment:
Could set a fixed number of attempts? It feels to me like if after like 5 consecutive attempts it is still failing, something else will be the problem, and then we don't just run into a timeout for the test.

Reply:
good point

Reply:
fixed